bin/cli.js

```diff
@@ -136,8 +164,12 @@ function invoke(env) {
       'Specify the stub extension (default js)'
     )
     .action(function(name) {
-      var instance = initKnex(env);
-      var ext = (argv.x || env.configPath.split('.').pop()).toLowerCase();
+      var instance = initKnex(env, commander);
+      var ext = (
```

Comment:
might make sense to extract to internal function with self-explanatory name or at least add comment with explanation

Reply:
Now using `initKnex(env, commander.opts())`. looks more explicit on its intention.